.java-version

```diff
@@ -0,0 +1 @@
+1.7
```

Comment:
What is this?

Reply:
this slipped through, I what I use to switch among JDK versions on my environment